vspace/src/main/java/edu/asu/diging/vspace/core/data/ModuleLinkRepository.java

```diff
@@ -3,17 +3,29 @@
 import java.util.List;
 
 import org.javers.spring.annotation.JaversSpringDataAuditable;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.stereotype.Repository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 
+import edu.asu.diging.vspace.core.model.IModule;
 import edu.asu.diging.vspace.core.model.impl.ModuleLink;
 import edu.asu.diging.vspace.core.model.IModuleLink;
 
 @Repository
 @JaversSpringDataAuditable
 public interface ModuleLinkRepository extends PagingAndSortingRepository<ModuleLink, String> {
+
+	
+    List<ModuleLink> findByModuleId(IModule module);
+       
+    @Query("select m from ModuleLink m where m.module.id = ?1")
+    List<ModuleLink> getModuleLinks(String moduleId);
```

Comment:
can't you just use the automatic query generation here? It's just a getByModule isn't it?

Reply:
used getByModule